src/modality-specific-files/eye-tracking.md

```diff
@@ -0,0 +1,233 @@
+so some terminology may be unfamiliar to researchers from other fields.
+This section adds clarifications to [Common Principles - Definitions](../common-principles.html)
+for the eye-tracking context.
+
+-   **Eye-tracker** - refer to the apparatus allowing the records of gaze position and/or pupil size.
+-   **Screen** - Video display used to present visual stimulation (for example projector, monitor, tablet).
+
+## Eye-tracking data
+
+```Text
+sub-<label>/
+    [ses-<label>/]
+        <datatype>/
+            sub-<label>[_ses-<label>][_task-<label>][_acq-<label>][_run-<index>]_eyetrack.<datatype>
+            sub-<label>[_ses-<label>][_task-<label>][_acq-<label>][_run-<index>]_eyetrack.json
+            sub-<label>[_ses-<label>][_task-<label>][_acq-<label>][_run-<index>]_events.json
+            sub-<label>[_ses-<label>][_task-<label>][_acq-<label>][_run-<index>]_events.tsv
+```
+
+The eye-tracking data files generally contain gaze position (x/y coordinates) and pupil size data.
+With Eye-Tracking-BIDS, we wish to promote the adoption of good practices i
+n the management of scientific data.
+Hence, the current emphasis of Eye-Tracking-BIDS is not to impose a generic data format for the modality,
+but to standardize the way data is stored in repositories.
+
+Eye-tracking data MUST be stored in the main data recording modality or `<datatype>` directory
+(for example `func`, `anat`, `dwi`, `meg`, `eeg`, `ieeg`, `beh`, `pet`...).
+The current version of this extension *does not* include a specification of the extension format
+and data formatting of recorded eye-tracking data. Thus, data must be stored in an open format
+(for example EDF file converted in ASCII `.asc` format or `.tsv`...) with the `_eyetrack` suffix.
+
+We encourage every user to put the raw data before conversion within the `/sourcedata` directory
+(for example put property EDF files in the sourcedata directory).
+
+The OPTIONAL [`task-<label>`](../appendices/entities.md#task) is used to indicate
+a task subject were asked to perform while eye-tracking records were obtained.
+Those labels MUST be consistent across subjects and sessions. For task based eye-tracking,
+a corresponding [task events](../modality-specific-files/task-events.md) file MUST be provided
+(please note that this file is not necessary for resting state tasks).
+
+The OPTIONAL [`acq-<label>`](../appendices/entities.md#acq) refers
+to a custom label the user MAY use to distinguish a different set of parameters used
+for acquiring the same modality. Acquisition labels corresponds mostly to imaging records
+(for example eye-tracking with fMRI) but can also be specified when combining eye-tracking and imaging methods.
+
+If more than one run of the same task and acquisition are acquired during the same session,
+the [`run-<index>`](../appendices/entities.md#run) entity MUST be used:
+_run-1, _run-2, _run-3, and so on. If only one run was acquired the run-<index> can be omitted.
```

Comment:
Those are redundant with the entity definition and should probably be removed.

Reply:
They are gone now.